scripts/vagrant/Vagrantfile

```diff
@@ -4,7 +4,13 @@
 Vagrant.configure("2") do |config|
 
   config.vm.synced_folder "../..", "/go/src/github.com/ligato/networkservicemesh"
+  config.vm.synced_folder ".", "/vagrant"
+  config.vm.allowed_synced_folder_types = [:virtualbox, :sshfs, :vmware]
```

Comment:
Please prioritize the higher performance options:
[:virtualbox, :vmware, :sshfs]